test/functional/mempool_reorg.py

```diff
@@ -27,82 +26,89 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
     def run_test(self):
+        wallet = MiniWallet(self.nodes[0])
+
         # Start with a 200 block chain
         assert_equal(self.nodes[0].getblockcount(), 200)
 
-        # Mine four blocks. After this, nodes[0] blocks
-        # 101, 102, and 103 are spend-able.
-        new_blocks = self.nodes[1].generate(4)
-        self.sync_all()
-
-        node0_address = self.nodes[0].getnewaddress()
-        node1_address = self.nodes[1].getnewaddress()
+        self.log.info("Add 4 coinbase utxos to the miniwallet")
+        # Block 76 contains the first spendable coinbase txs.
+        first_block = 76
+        wallet.scan_blocks(start=first_block, num=4)
 
         # Three scenarios for re-orging coinbase spends in the memory pool:
-        # 1. Direct coinbase spend  :  spend_101
-        # 2. Indirect (coinbase spend in chain, child in mempool) : spend_102 and spend_102_1
-        # 3. Indirect (coinbase and child both in chain) : spend_103 and spend_103_1
+        # 1. Direct coinbase spend  :  spend_1
+        # 2. Indirect (coinbase spend in chain, child in mempool) : spend_2 and spend_2_1
+        # 3. Indirect (coinbase and child both in chain) : spend_3 and spend_3_1
         # Use invalidatblock to make all of the above coinbase spends invalid (immature coinbase),
         # and make sure the mempool code behaves correctly.
-        b = [self.nodes[0].getblockhash(n) for n in range(101, 105)]
+        b = [self.nodes[0].getblockhash(n) for n in range(76, 80)]
```

Comment:
```suggestion
        b = [self.nodes[0].getblockhash(n) for n in range(first_block, first_block+4)]
```

Reply:
Any reason you didn't fix this as well on the latest force push?

Reply:
sorry i didn't see this comments, i'll fix it now!

Reply:
just pushed it. Thanks for the review